duckdb_capi/duckdb.h

```diff
@@ -1341,7 +1341,7 @@ DUCKDB_API duckdb_date duckdb_to_date(duckdb_date_struct date);
 Test a `duckdb_date` to see if it is a finite value.
 
 * @param date The date object, as obtained from a `DUCKDB_TYPE_DATE` column.
-* @return True if the date is finite, false if it is ±infinity.
+* @return true if the date is finite, false if it is ±infinity.
```

Comment:
The c++ type `bool` hold `true` or `false`, and other parts of duckdb use the proper case when documenting bools, e.g.:

https://github.com/duckdb/duckdb/blob/1f8b6839ea7864c3e3fb020574f67384cb58124c/src/include/duckdb/storage/index.hpp#L48-L49